src/components/editions/headerImage.tsx

```diff
@@ -12,6 +13,15 @@ import type { FC } from 'react';
 
 const styles = css`
 	margin: 0;
+	position: relative;
+	div {
+		background-color: ${brandAltBackground.primary};
+		position: absolute;
+		bottom: 0;
+		left: 0;
+		${body.medium({ fontWeight: 'regular' })};
+		padding: 0px 5px 3px 5px;
```

Comment:
Prefer `remSpace` over `px`.

Reply:
👍

Reply:
Cant quite get the precise px conversion with remSpace as `remSpace[1]` = 0.25 and `remSpace[1]` = 0.5, I would have to use `rem` to specify 0.3125 for 5px

Reply:
I'd suggest just converting to Source values:

```suggestion
		padding: 0px ${remSpace[1]} ${remSpace[1]} ${remSpace[1]};
```

and seeing how that looks.